FlagsmithClient/Classes/Flagsmith.swift

```diff
@@ -42,8 +45,24 @@ public class Flagsmith {
     set { analytics.flushPeriod = newValue }
     get { analytics.flushPeriod }
   }
+  
+  /// Default flags to fall back on if an API call fails
+  public var defaultFlags: [Flag] = []
+
+  /// Cache to use when enabled, defaults to the shared app cache
+  public var cache: URLCache = URLCache.shared
+
+  /// Use cached flags as a fallback?
+  public var useCache: Bool = false
+
+  /// TTL for the cache in seconds, default of 0 means infinite
+  public var cacheTTL: Double = 0
 
-  private init() {}
+  /// Skip API if there is a cache available
+  public var skipAPI: Bool = false
```

Comment:
Not sure how feasible this is here but should we push all of these into a config dataclass for example?